oe_paragraphs.post_update.php

```diff
@@ -200,3 +201,36 @@ function oe_paragraphs_post_update_10006(array &$sandbox): void {
   $field->set('settings', $settings);
   $field->save();
 }
+
+/**
+ * Updates the allowed_values management of field_oe_icon.
+ */
+function oe_paragraphs_post_update_10007(array &$sandbox): void {
```

Comment:
Was this update tested? Because I get this:
```
francesco in ~/Projects/web/oe_paragraphs <git:OEL-759>
[18:36:12] ❯❯❯ docker-compose exec web ./vendor/bin/drush updb -y
 --------------- ----------- ------------- ------------------------------
  Module          Update ID   Type          Description
 --------------- ----------- ------------- ------------------------------
  oe_paragraphs   10007       post-update   Updates the allowed_values
                                            management of field_oe_icon.
 --------------- ----------- ------------- ------------------------------


 // Do you wish to run the specified pending updates?: yes.

>  [notice] Update started: oe_paragraphs_post_update_10007
>  [error]  A list field 'field_oe_icon' with existing data cannot have its keys changed.
>  [error]  Update failed: oe_paragraphs_post_update_10007
 [error]  Update aborted by: oe_paragraphs_post_update_10007
 [error]  Finished performing updates.
```

Reply:
We needed to avoid problem that is detected when there's data (so calling to options.module). So, we added a temporary storage and then restore data. Please check it.